webapi/getfeeaddress.go

```diff
@@ -41,25 +41,14 @@ func getNewFeeAddress(db *database.VspDatabase, addrGen *addressGenerator) (stri
 }
 
 func getCurrentFee(dcrdClient *rpc.DcrdRPC) (dcrutil.Amount, error) {
-	bestBlock, err := dcrdClient.GetBestBlockHeader()
-	if err != nil {
-		return 0, err
-	}
-	sDiff, err := dcrutil.NewAmount(bestBlock.SBits)
-	if err != nil {
-		return 0, err
-	}
-	relayFee, err := dcrutil.NewAmount(relayFee)
-	if err != nil {
-		return 0, err
-	}
 
-	fee := txrules.StakePoolTicketFee(sDiff, relayFee, int32(bestBlock.Height),
-		cfg.VSPFee, cfg.NetParams)
+	var ticketInfo dcrdtypes.InfoChainResult
+	relayFee, err := dcrutil.NewAmount(ticketInfo.RelayFee)
```

Comment:
This file seems to be the only file in this PR that relates to the proposed solution. Please clean up.
Also, `relayFee` will always be 0.